consensus/merge/src/test/java/org/hyperledger/besu/consensus/merge/blockcreation/MergeCoordinatorTest.java

```diff
@@ -327,7 +333,7 @@ public void ancestorIsValidTerminalProofOfWork() {
 
   @Test
   public void ancestorNotFoundValidTerminalProofOfWork() {
-    final long howDeep = MergeCoordinator.MAX_TTD_SEARCH_DEPTH + 1;
+    final long howDeep = MergeCoordinator.MAX_TTD_SEARCH_DEPTH + 2;
```

Comment:
Just for my understanding: Why do we need + 2 here instead of + 1?

Reply:
Because we start the search from the parent so we need the chain depth to be at longer than max_depth + 1, so +2 ;)